gradle/dependencies.gradle

```diff
@@ -1,36 +1,36 @@
 ext {
     androidVersions = [
-            minSdkVersion    : 14,
-            targetSdkVersion : 28,
-            compileSdkVersion: 28,
+            minSdkVersion    : 21,
```

Comment:
deserves changelog entry, doesn't it?

Reply:
as a stand-alone library, this minSdkVersion bump seems to be a major breaking change. I'd suggest to keep the same 14 as before.

Reply:
Well, ok